frontend/src/metabase/visualizations/components/Visualization/Visualization.tsx

```diff
@@ -344,22 +526,29 @@ class Visualization extends PureComponent {
       actionButtons,
       className,
       dashcard,
-      getHref,
+      errorIcon,
       errorMessageOverride,
-      showTitle,
-      isDashboard,
-      width,
-      height,
+      expectedDuration = 0,
```

Comment:
Not sure if the `0` is the proper fallback value

Reply:
I removed the default value from here, but inside the `LoadingView` component i added the handling of the nullable `expectedDuration` value